src/foobar/enums.py

```diff
@@ -4,3 +4,9 @@
 class PurchaseStatus(enum.Enum):
     FINALIZED = 0
     CANCELED = 1
+
+
+class TrxType(enum.Enum):
+    CORRECTION = 0
+    DEPOSIT = 1
+    WITHDRAW = 2
```

Comment:
`WITHDRAW` should probably be changed too from being a verb to a noun: `WITHDRAWAL`.